build.gradle

```diff
@@ -3,10 +3,13 @@
 buildscript {
     repositories {
         jcenter()
+        maven {
+            url 'https://maven.google.com/'
```

Comment:
You can use google() now in Gradle

Reply:
Added!

Reply:
commit b1e33a5b4f11f184227741d8d32593267cbb4c73